shop/src/pages/admin/settings/checkout/_Taxes.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect } from 'react'
+import fbt from 'fbt'
+import get from 'lodash/get'
+
+import CountryTaxEntry from './_CountryTaxEntry'
+
+const ManualTaxes = ({ state, setState }) => {
+  const taxRates = get(state, 'taxRates')
+
+  useEffect(() => {
+    if (!taxRates || !taxRates.length) {
+      setState({
+        taxRates: [{}]
+      })
+    }
+  }, [taxRates])
+  return (
+    <div className="manual-taxes">
+      <div className="form-group">
+        <label className="mb-0">
+          <fbt desc="Taxes">Taxes</fbt>
+        </label>
+        <div className="desc mb-3">
+          <fbt desc="admin.settings.checkout.taxesDesc">
+            Charge taxes on your orders. By enabling this features, you
+            acknowledge that it is up to you and a tax professional to determine
+            when and how much tax to collect and remit, and if this feature is
+            right for you.
+          </fbt>
+        </div>
+        {/* <label className="form-check mb-3 font-weight-normal">
```

Comment:
is this commented out intentionally?

Reply:
Yes, we actually don't store the user's address until the later steps of checkout (which also gets cleared after placing the order), so we cannot show the product prices with taxes. I've raised this with @gullible1, will have to come back to this once a decision is taken regarding this.

Reply:
Sounds good. Maybe add a comment in there so that when we look at this code again we have the context as to why it was commented out.

Reply:
Sure, will do.